code/game/objects/effects/decals/crayon.dm

```diff
@@ -823,56 +1010,121 @@ obj/item/scroll/attackby(obj/item/I, mob/living/carbon/human/M)
 				T.vessel.remove_reagent("blood", ((T.get_blood_volume() * 0.01) - bloodpercent) * T.species.blood_volume)
 			else T.vessel.add_reagent("blood", (bloodpercent - (T.get_blood_volume() * 0.01)) * T.species.blood_volume)
 			to_chat(T, "<span class='warning'>You feel extremly woozy and light headed for a second. It partially recovers.</span>")
+			M.sanity.changeLevel(-5) //Good deads always get punished (but only if we successfully cast the spell!)
 
-		M.sanity.changeLevel(-5) //Good deads always get punished
 	//caster blood handling below
 	to_chat(M, "<span class='warning'>The sound of a heart beat fills the air around you.</span>")
+	playsound(loc, 'sound/effects/singlebeat.ogg', 80)
 	if(M.get_blood_volume() < bloodpercent)
 		M.vessel.add_reagent("blood", (bloodpercent - (M.get_blood_volume() * 0.01)) * M.species.blood_volume)
 	else M.vessel.remove_reagent("blood", ((M.get_blood_volume() * 0.01) - bloodpercent) * M.species.blood_volume)
 	B.remove_self(min(20 * targets.len, 80))
 	return
 
-/obj/effect/decal/cleanable/crayon/proc/scroll_spell(mob/living/carbon/human/M) //able to be cast by all. But only filled out by scribes.
+// Baptism: High blood cost, to invoke a bloody basin in which to soak one's hands for writing in blood
+// This makes it so that you don't need to gib additional creatures to write each time
+// TODO: MORE CRAYON CULT BASED FURNITURE, CHANDELIERS?
+/obj/effect/decal/cleanable/crayon/proc/basin_spell(mob/living/carbon/human/M)
+	var/datum/reagent/organic/blood/B = M.get_blood()
+	for(var/obj/structure/reagent_dispensers/watertank/W in oview(1)) // Must be on the spell circle
+		to_chat(M, "<span class='info'>Thunder crackles as a miniature cloud of nothingness manifests itself. Blood begins pouring down, forming an omnious obsidian basin beneath it...</span>")
+		B.remove_self(100) // Basically pouring your blood into a container, insane
+		M.sanity.breakdown(TRUE) // If your blood got sucked and poured into a container you too would freak out
+		M.sanity.changeLevel(-50)
+		var/obj/structure/sink/basion/crayon/N = new /obj/structure/sink/basion/crayon
```

Comment:
i thought this was going to be a debug obj

Reply:
It's pretty handy and non-random unlike fucking gibs which give you always a RANDOM AMOUNT between 1 and 5 and you have to PRAY that you don't run out of of blood while writing on the floor + the double-dip bug nixing your ability to write in blood on the floor
It also has a snowflake sprite now